src/esm_tests/cli.py

```diff
@@ -25,6 +25,12 @@ def main():
         filter={"": "WARNING", "esm_tests": "DEBUG"},
         format="<level>{message}</level>",
     )
```

Comment:
It might be a good idea to initialize the logging level. Eg `"INFO"` or `"DEBUG"` in `logger.add`. The default is DEBUG in loguru.
Eg.
```Python
logging_level = "INFO"
logger.add(..., level=logging_level, ...) 
```

Reply:
That it's done on the `cli.py` already.

Reply:
I keep the default format for now on levels that are not `"DEBUG"`. I won't add this now, as I am trying to speedup and I need this for testing levante.